src/app/course/event/event-row/event-row.component.html

```diff
@@ -4,10 +4,10 @@
         </ng-container>
         <ng-container *ngIf="event.type === 'EXAM'">Exam</ng-container>
     </div>
-    <div class="flex flex-grow items-center">
+    <div class="flex items-center sm:flex-grow lg:min-w-[21rem]">
```

Comment:
I don't understand, why did you have to this?

Reply:
This is so the view doesn't look weird on mobile